honeybee_energy/hvac/heatcool/_base.py

```diff
@@ -41,8 +42,9 @@ class _HeatCoolBase(_TemplateSystem):
         * sensible_heat_recovery
         * latent_heat_recovery
         * schedules
+        * properties
     """
-    __slots__ = ()
+    __slots__ = ('_properties')
```

Comment:
whoops - missed that. Actually: I wonder if/how I should add some tests for that as well? I did run pytest and got all passing, and coverage does show this line as getting covered too? But didn't get any errors anyplace (though clearly it is). 

I'd be happy to add tests to the set to catch these in the future. Not sure why this doesn't throw any error though or how to write the test for something like this? maybe adding a test for `__dict__` (to make sure its does NOT exist) after object instantiation?

Reply:
Hey @PH-Tools ,

This is a tough thing to test for since, if you do it incorrectly like this, it just results in `__slots__` not working the way that it should to reduce the memory. We'll just have to remember to be good about it and perform code reviews.